### ISSUE TITLE
[Feature] Link to Billing in menu

### ISSUE THREAD

**Issue by maphew** [lencx/ChatGPT#252]
### Feature description

It would be nice if the app could show current Usage and Billing status.

The easy method would be to simply to add menu links:
* https://beta.openai.com/account/usage
* https://beta.openai.com/account/billing/overview

More advanced (and nicer/cooler) would be to use the API. which chatgpt says is a matter of:
 
> ...you can check on your token usage by using the OpenAI API, by sending a GET request to the /api/usage endpoint with your API key in the header.

### Motivation

The motivation is to keep track of my OpenAI token usage the same way I know how much fuel I have remaining in my car and battery left in my mobile and laptop devices: how far can I go? Do I need to be smarter about what questions I'm asking in order not to run out?

**Comment by Manamama** [comment]
Hm. ChatGPT is "free" (sponsored) as of today, yet [see the plans](https://github.com/lencx/ChatGPT/issues/239#issuecomment-1404832731). 
AFAIK, https://beta.openai.com/account/usage is another engine: I have been testing it since SEP 2022 and ChatGPT is more forgiving with "throwaway" email addresses, which had been blocked by beta.openai around NOV 2022.  

In short, IMHO it is not needed yet here, still as of today's status quo.
